PizzaOvenUI/BackEndConnection.qml

```diff
@@ -367,7 +367,8 @@ Item {
             if (msg.data.time && msg.data.time>=0 && msg.data.time<=65535) {
                 cookTime = msg.data.time;
                 sendMessage("WriteTimerSettingResponse id " + msg.data.id + " result success");
-                utility.findPizzaTypeFromSettings();
+                // The following line was causing strange things to happen. Removing it seems to fix the issue.
+                // utility.findPizzaTypeFromSettings();
```

Comment:
I don't think we actually want to try to find the pizza type here. It will almost always return "custom" because only the cookTime is specified here and the function requires all settings to match before loading a cycle (upper/lower/cook time). I'm going to re-test to confirm this.

Reply:
Also figured out how to make this work.